step.yml

```diff
@@ -220,9 +228,21 @@ outputs:
         Public Install Page URLs by the artifact's file path.
         Only set it if the *Enable public page for the App?* option was *enabled*.
 
-        The format is `KEY1=>VALUE|KEY2=>VALUE`
+        The default format is `KEY1=>VALUE|KEY2=>VALUE` but is controlled by the `public_install_page_url_map_format` input
 
         Examples:
 
         - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
         - $BITRISE_DEPLOY_DIR/android_app.apk=>https://android_app/public/install/page|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
+  - BITRISE_PERMANENT_DOWNLOAD_URL_MAP:
+    opts:
+      title: Map of filenames and Permanent Download URLs
+      description: |-
+        Permanent Download URLs by the artifact's file path.
+
```

Comment:
If there are multiple artifact URLs, you can customize their format.

Reply:
This after line 241 or instead something?

Reply:
"If there are multiple artifact URLs, you can customize their format." should come before the example you provided below. It can be a continuation of line 242 or right after it, but I'd definitely add it before the examples and the default format. :)